### PR TITLE
[PRM-1397] Changed to UseAPI1812Model feature switch for Calculation Controller

### PR DESCRIPTION
**Pull request by JBlacklaws**

### CODE REVIEW

File: app/controllers/CalculationController.scala [hunk 1]
```diff
@@ -34,7 +34,7 @@ import utils.Logger.logger
 import utils.{CurrencyFormatter, EnrolmentKeys}
 import viewmodels.CalculationPageHelper
 import models.point.PointStatusEnum
-import config.featureSwitches.{CallAPI1812ETMP, FeatureSwitching}
+import config.featureSwitches.{CallAPI1812ETMP, FeatureSwitching, UseAPI1812Model}
```
Comment (ruthdas03) [inline app/controllers/CalculationController.scala:37]: Can we not remove the unused import CallAPI1812ETMP

**Review by ruthdas03 — COMMENTED** [review]
Please fix the tests.

**Comment by LewisMitchell** [comment]
I don't think the tests are failing because of the changes. I think it is a transient issue 😄

**Comment by LewisMitchell** [comment]
test this please

**Pull request merged**